external/kafka/src/main/scala/org/apache/spark/streaming/kafka/DirectKafkaInputDStream.scala

```diff
@@ -158,4 +166,37 @@ class DirectKafkaInputDStream[
     }
   }
 
+  private[streaming]
+  class DirectKafkaStreamingListener extends StreamingListener {
+
+    val offsetCommitEnabled = context.conf.getBoolean("spark.streaming.kafka.offsetCommit", false)
```

Comment:
This is a new configuration key, right?

If I'm a new user and see a name like offsetCommit, I'm likely to be confused as to whether the offsets in question are being used as the system of record (they aren't), and whether this is something I have to enable for durability (it isn't).